lib/active_model/serializer/adapter/attributes.rb

```diff
@@ -2,42 +2,73 @@ module ActiveModel
   class Serializer
     module Adapter
       class Attributes < Base
+        attr_accessor :_nested_associations
+
+        def initialize(serializer, options = {})
+          options ||= {}
+
+          self._nested_associations = ActiveModel::Serializer::Utils
+            .include_args_to_hash(options[:include])
+
+          super(serializer, options)
+        end
+
         def serializable_hash(options = nil)
           options ||= {}
+
+          unless _nested_associations.present?
+            # not set in the initializer when rendered from a controller
```

Comment:
The **only** options passed from the controller to the adapter are defined in SerializableResource.  Everything else goes to the Serializer